raiden/ui/app.py

```diff
@@ -198,12 +163,14 @@ def run_app(
     if not parsed_eth_rpc_endpoint.scheme:
         eth_rpc_endpoint = f'http://{eth_rpc_endpoint}'
 
-    web3 = _setup_web3(eth_rpc_endpoint)
-    node_network_id, known_node_network_id = setup_network_id_or_exit(config, network_id, web3)
+    web3 = Web3(HTTPProvider(eth_rpc_endpoint))
+    check_ethereum_version(web3)
+    check_network_id(network_id, web3)
+    config['chain_id'] = network_id
```

Comment:
moving the assignments up reduces the number of dependencies for these check functions.